### ISSUE TITLE
Separate `app-starter` from examples

### ISSUE THREAD

**Issue by JakobMiksch** [meggsimum/wegue#274]
relates to #265 and #272

quote from @fschmenger in #265:

> I propose to resemble the folder structure we have under app-starter for all the specific examples. app-starter itself will be stripped down to only contain a single app-conf (maybe even just the minimal one?!). All specific static resources (e.g. shops-dannstadt.geojson, 2012_Earthquakes_Mag.kml, etc.) will be moved to the respective examples they are used for. Then a user could run the examples e.g. by an app-init script in a similar fashion like we do now. Quite a bunch of work but probably more future proof :)

**Comment by fschmenger** [comment]
This should be on hold, until we have implemented #276. If you agree mark it as a blocker please.

**Comment by JakobMiksch** [comment]
currently we do not have the concept of "blocker" tags yet. But we can introduce that

**Comment by fschmenger** [comment]
My bad. I thought Github had some dependency management between issues, such as you could mark "A as blocked by B". But apparently that's not the case. "Blocker" is a little bit confusing, I would rather label this one "blocked" because #276 is the potential blocker :)

**Comment by JakobMiksch** [comment]
I changed it to 'blocked'
